src/App.js

```diff
@@ -16,46 +32,101 @@ const generateSquares = () => {
     for (let col = 0; col < 3; col += 1) {
       squares[row].push({
         id: currentId,
-        value: '',
+        value: "",
       });
       currentId += 1;
     }
   }
-
   return squares;
 }
 
 const App = () => {
-
   const [squares, setSquares] = useState(generateSquares());
+  const [currentPlayer, setPlayer] = useState(PLAYER_1)
+  const [winner, setWinner] = useState(null);
+  const [count, setCurrentCount] = useState(0)
 
   // Wave 2
   // You will need to create a method to change the square 
-  //   When it is clicked on.
-  //   Then pass it into the squares as a callback
-
+  //  When it is clicked on.
+  //  Then pass it into the squares as a callback
+  const onClickCallback = (id) => {
+    if (winner === "PLAYER_1" || winner === "PLAYER_2") return
```

Comment:
This doesn't seem to be quite working as when the player wins you can keep clicking.

```suggestion
    if (winner !== null) return
```